src/env.spec.ts

```diff
@@ -379,6 +379,7 @@ describe("setEnvConfiguration", () => {
         enableSourceCodeIntegration: true,
       },
       handlers,
+      "nodejs10.x",
```

Comment:
I just needed to add a runtime here since the function now takes in runtime and I picked the first one I saw. It's not actually doing anything since it isn't dotnet. I still need to fix and add more tests though so I'll update this